### PR TITLE
Ch test signup jwt 158121598

### PR DESCRIPTION
**Pull request by ddouglasz**
### What does this PR do?
- This pull request provides a test for a successful signup with a JWT response.
### Description of Task to be completed?
- When a user signs up successfully, there should be a token generated for that session.
### How should this be manually tested?
- Run  *$npm test*  in the terminal
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/157801291
### Screenshots (if appropriate)
N/A
### Questions:
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "passport-jwt": "^4.0.0",
     "passport-local": "~0.1.6",
     "passport-twitter": "^1.0.4",
+    "request": "^2.87.0",
```
Comment (fortunee) [inline package.json]: Hi @ddouglasz I think `supertest` does what this package does for you. Why do you have it installed?

File: test/endpoints/auth.spec.js [hunk 2]
```diff
@@ -19,3 +20,21 @@ describe('Authentication', () => {
...
+  it('Should return a JWT upon user signup', (done) => {
+    request
+      .post('/api/auth/signup')
+      .send({
+        name: 'stevenson',
+        email: 'simonsinec@gmail.com',
+        password: 'password567'
+      })
+      .end((err, res) => {
+        res.body.message.should.equal('Signed up successfully, please check email for activation link');
```
Comment (fortunee) [inline test/endpoints/auth.spec.js]: Are you testing that the user receives a JWT or that he receives an activation link? @ddouglasz

**Review by fortunee — COMMENTED** [review]
Hi, @ddouglasz Please go through my comment and let's have a call to discuss more.

File: package.json (hunk 1 above)
Comment (ddouglasz) [reply package.json]: Thank you @fortunee. I have fixed it. I took it off.

**Pull request closed without merging**